client/src/App.jsx

```diff
@@ -1,15 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Header from './Header.jsx';
+import Main from './Main.jsx'
+import sampleSources from '../../db/testData.js';
 import styles from './styles.css';
```

Comment:
I wonder if moving this to the index.html file will solve the problem?  Thoughts?